app/src/main/java/org/dhis2/animations/CarouselViewAnimations.kt

```diff
@@ -1,11 +1,10 @@
 package org.dhis2.animations
 
 import android.view.animation.DecelerateInterpolator
-import org.dhis2.uicomponents.map.views.CarouselView
 
 class CarouselViewAnimations {
 
-    fun initMapLoading(view: CarouselView) {
+    fun initMapLoading(view: org.dhis2.maps.views.CarouselView) {
```

Comment:
This could be a good time to use kotlin typealiases
`
typealias Carousel = org,dhis2.maps.views.CarouselView
...
fun initMapLoading(view: Carousel) {}
`